docs/index.rst

```diff
@@ -1,23 +1,108 @@
-.. pluggy documentation master file, created by
```

Comment:
@nicoddemus sure if you'd like?

Do you mean more like `pytest`'s [index.rst](https://github.com/pytest-dev/pytest/blob/master/doc/en/index.rst) and docs [homepage](http://docs.pytest.org/en/latest/) which seems to double as the github readme?

So you want me to strip the `User Guide` section (with its TOC) and have a docs section which links to individual .rst documents?